man/osg-incommon-cert-request.1

```diff
@@ -79,6 +82,14 @@ hostname04.yourdomain hostname05.yourdomain
 .RE
 .fi
 .RE
+.PP
+To provide alternative organization and department codes for the InCommon Certificate Service. 
+.PP
+.RS
+.nf
+osg-incommon-cert-request -O 4567,8912 -u incommonuser -c certpath.pem -k keypath.pem -H hostname.yourdomain
```

Comment:
Lines should be 80 characters or shorter.  You can break them up with a backslash, as is done in another example.  I see now that there are several other lines that are also longer than 80 characters, please break up those example commands too.  There's also a hostfilepath.txt example line that's too long; just choose shorter example names.

Reply:
Fixed.